### ISSUE TITLE
How to restart playlist settings on restart?

### ISSUE THREAD

**Issue by srezasm** [mpv-player/mpv#9133]
Is there any config flag to enable restart playlist video volume and speed on each mpv restart?

**Comment by guidocella** [comment]
What do you mean by restart playlist setting?

**Comment by srezasm** [comment]
@guidocella  The settings like volume and speed that sticks to each playlist

**Comment by guidocella** [comment]
I still don't understand what you mean. If you mean saving options with quit-watch-later, by default the volume is saved and not the speed, and these can be changed with `--watch-later-options` if you use the latest master.

**Comment by hooke007** [comment]
No, `--watch-later-options` is only used for every file, so far no way to save the global options (also including playlists).

**Comment by srezasm** [comment]
@hooke007 Then I think this thread should be labeled as a feature request
